debian/rules

```diff
@@ -1,12 +1,34 @@
 #!/usr/bin/make -f
 # vi: ts=8 sw=8 noet
 
-export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+export DEB_BUILD_MAINT_OPTIONS  = hardening=+all
+
+# USE CJSON_SO / USE_LIST_SO:
+# If set to 1 it will not use the shipped version, but the shared library
+# installed via the corresponding lib[cjson|list]dev packages
+export USE_CJSON_SO = 1
```

Comment:
this is generally fine; but short reminder but for buster we cannot use the cjson package, because of an unfixed bug

Reply:
Will fix this in Makefile

Reply:
I'm using libcjson1_1.7.14-1 (i.e. the bullseye version, buster is 1.7.10-1.1) on buster build environment too.

Would that fix this?

Reply:
one could think that, but it's not the case, because the problem is not with building, but when running the agent, so what matters is the version the user has installed and on buster this will be 1.7.10-1.1 which will be problematic

Reply:
So, for buster, compiling with USE_CJSON_SO = 0 would fix it?

Reply:
yes

Reply:
ok; added to our -2.0 cow-build-oidc-all-2.0sh build script.